### ISSUE TITLE
Use entities in dropdown choice arguments

### ISSUE THREAD

**Issue by FieldofClay** [OliveTin/OliveTin#247]
**Is your feature request related to a problem? Please describe.**
It would be awesome to use the new entities functionality to populate the choices for an action.

**Describe the solution you'd like**
Something like:

``` yaml
 - title: Restart Docker Container
   icon: restart
   shell: docker restart {{ container }}
   arguments:
     - name: container
       title: Container name
       entity: container
       choices:
         - value: {{ container.name }}
```

**Describe alternatives you've considered**
The dashboard functionality works, but having a single "restart" action with a dropdown would be less cluttered than a dozen "restart container" actions.

**Comment by jamesread** [comment]
Hey @FieldofClay , thanks for taking the time to write up this feature request.

This is entirely logical, I'm not sure why I didn't think of this before. I'm happy to implement this feature. 

I'd need to look into how the arguments list is populated at the moment to remind myself of how that part of the code works, I want to make sure that the implementation of these doesn't make things too messy. Adding an `entity` field to the argument as well, might be the way to go.  For example: 

```yaml
actions: 
  - title: restart container
    shell: docker restart {{ containerToRestart}} 
    arguments: 
      - name: containerToRestart 
        entity: container
        title: {{ container.name }} 
```

**Comment by jamesread** [comment]
This has now been implemented in the code and will be available in the next release of OliveTin, possibly tonight, possibly the next few days. 

It is documented here; http://localhost/docs.olivetin.app/docs/arg-dropdowns.html#args-dropdown-entities

**Comment by jamesread** [comment]
Released in 2024.03.24.
